app/views/layouts/_phase_banner.html.haml

```diff
@@ -1,7 +1,10 @@
-.govuk-phase-banner
+.govuk-phase-banner{ class: ENV['ENV'] }
   %p.govuk-phase-banner__content
     %strong.govuk-tag.govuk-phase-banner__content__tag
-      = t('generic.alpha')
+      - if %w(uat staging dev).include? ENV['ENV']
+        = ENV['ENV']
+      - else
+        = t('generic.alpha')
```

Comment:
Should we set a new variable to the value of `ENV['ENV']` at the top and use that throughout the page.  If none exists (local), we should set it to "local" (or similar).  That way we can also set colours for those running locally.

Reply:
I have created app_environment in application_helper.